재호/2667.py

```diff
@@ -0,0 +1,56 @@
+# https://www.acmicpc.net/problem/2667
+
+
+def is_valid_coordinate(size, y, x):
+    return 0 <= y and y < size and 0 <= x and x < size
```

Comment:
```suggestion
    return 0 <= y < size and 0 <= x < size
```
파이썬은 이런 표현이 가능한걸로 알고있어요!